src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,72 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (buttonPage: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const numOfPage = Math.ceil(total / perPage);
+  const buttonsOnPage = getNumbers(1, numOfPage);
+
+  const choosePage = (elementButton: number) => {
+    if (elementButton !== currentPage) {
+      onPageChange(elementButton);
+    }
+  };
+
+  return (
+    <ul className="pagination">
+      <li className={cn('page-item', { disabled: currentPage === 1 })}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1}
+          onClick={() => currentPage !== 1 && choosePage(currentPage - 1)}
```

Comment:
condition is redundant since you disable a button with this condition
```suggestion
          onClick={() => choosePage(currentPage - 1)}
```